File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # Promt user for height
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break

    # Print blocks
    print_blocks(height)


# Function that prints a pyramid with a given height
def print_blocks(height):
    count = 1
    for i in range(height):
        print(" " * (height - count), end="")
        print("#" * count, end="")
        print(" " * 2, end="")
        print("#" * count)
        count += 1


main()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

// Defining functions before main
int space(int n);
int hash(int n);

int main(void)
{
    // Promt user for height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print hashes with promted height
    for (int i = 1; i < height + 1; i++)
    {
        space(height - i);
        hash(i + 1);
        space(2);
        hash(i + 1);
        printf("\n");
    }
}

// Function that prints n spaces
int space(n)
{
    for (int i = 1; i < n + 1; i++)
    {
        printf(" ");
    }
    return 1;
}

// Function that prints n hashes
int hash(n)
{
    for (int i = 1; i < n; i++)
    {
        printf("#");
    }
    return 1;
}